engine/crates/federated-graph/src/render_sdl.rs

```diff
@@ -184,3 +195,44 @@ fn render_field_arguments(args: &[FieldArgument], graph: &FederatedGraph) -> Str
         out
     }
 }
+
+struct SelectionSetDisplay<'a>(&'a FieldSet, &'a FederatedGraph);
```

Comment:
Not important, but since this is printing a FieldSet should it be named`FieldSetDisplay`?